frameworks/cassandra/universe/config.json

```diff
@@ -468,6 +468,32 @@
             "type": "application/x-zone-constraints+json"
           }
         },
+        "external_volume": {
+          "type": "object",
+          "description": "The Cassandra external volume configuration.\nOnly Portworx external volumes are supported.",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "If true, external profile will be used.",
+              "default": false
+            },
+            "portworx_volume_options": {
+              "type": "string",
+              "default": "size=10",
+              "description": "Volume options."
+            },
+            "use_service_path": {
```

Comment:
I think we can eliminate this option and make things cleaner.

if `volume-name` == "": retreive service path.
else use `volume_name`

In the code, we should be able to return an `Optional<String>` and check if the Optional is empty.